reference-files/token-map.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This isn't necessarily something that I would say is scalable long term because it could get pretty large. That said, it's a good starting point in how to go from css -> json. Most of the libraries I could find go the other way.

Reply:
Could this file start out with all the "top tokens" so that users don't have to go searching for the key side? Values can be `""`.

Reply:
Great point, and something that allows the `topTokens.txt` file to be put to actual use.

Reply:
Ported it over